_posts/hannahlwang/2016-02-02-hannah-ch5.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: post
+author: hannahlwang
+title: "Hannah's Chapter 5 Exercises"
+---
+
+Exercise 1:
+
+<iframe src="https://trinket.io/embed/python/4474baa531" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+Exercise 2:
+
+<iframe src="https://trinket.io/embed/python/84fd92dbdc" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+The exercises for this week were relatively straightforward, since the readings did a pretty good job of explaining definite loops using "for". The only tricky parts for me were setting the original "count", "total", "largest", and "smallest" values outside the loop, the putting the try/except inside the loop, so that the program would keep asking for input even after a bad input.
```

Comment:
Try your programs with two digit inputs like `11`- the sums are wrong.  Can you figure out why?